header-moh.php

```diff
@@ -21,40 +21,44 @@
 <title><?php wp_title( '|', true, 'right' ); ?></title>
 <link rel="profile" href="http://gmpg.org/xfn/11" />
 <link rel="pingback" href="<?php bloginfo( 'pingback_url' ); ?>" />
-<link href='http://fonts.googleapis.com/css?family=Open+Sans:300,400,400italic,600,600italic,700,700italic' rel='stylesheet' type='text/css'>
-<script src="https://code.jquery.com/jquery-1.11.3.min.js"></script>
```

Comment:
Is jquery unused on this site? Or included in some other way?

Reply:
jquery library is loaded in functions.php, the reference is not needed in theme templates